### DISCUSSION TITLE
Headers required for Shared Array Buffer

### DISCUSSION THREAD

**Discussion by patrickcorrigan** [ionic-team/capacitor#7553] [Q&A]
Hi I'm currently using capacitor to create an iOS app for my web app. Some features of my app require using Shared Array Buffer https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer which requires the document to have the following headers

```
Cross-Origin-Opener-Policy: same-origin
Cross-Origin-Embedder-Policy: require-corp
```

Is it possible to set custom headers like this in the capacitor web server?

**Comment by lumenwrites** [comment]
Hi! Did you find a solution to this?
